service-pdf/src/Opg/Lpa/Pdf/AbstractLp1.php

```diff
@@ -259,29 +341,50 @@ abstract protected function populatePageSix(PrimaryAttorneyDecisions $primaryAtt
-            //  If there is a person to notify for this index then render the details
+            // If there is a person to notify for this index then render the details
             if (array_key_exists($i, $peopleToNotify)) {
                 $personToNotify = $peopleToNotify[$i];
 
-                $this->setData('lpa-document-peopleToNotify-' . $i . '-name-title', $personToNotify->name->title)
-                     ->setData('lpa-document-peopleToNotify-' . $i . '-name-first', $personToNotify->name->first)
-                     ->setData('lpa-document-peopleToNotify-' . $i . '-name-last', $personToNotify->name->last)
-                     ->setData('lpa-document-peopleToNotify-' . $i . '-address-address1', $personToNotify->address->address1)
-                     ->setData('lpa-document-peopleToNotify-' . $i . '-address-address2', $personToNotify->address->address2)
-                     ->setData('lpa-document-peopleToNotify-' . $i . '-address-address3', $personToNotify->address->address3)
-                     ->setData('lpa-document-peopleToNotify-' . $i . '-address-postcode', $personToNotify->address->postcode);
+                $this->setData(
+                    'lpa-document-peopleToNotify-' . $i . '-name-title',
+                    $personToNotify->name->title
+                );
+                $this->setData(
+                    'lpa-document-peopleToNotify-' . $i . '-name-first',
+                    $personToNotify->name->first
+                );
+                $this->setData(
+                    'lpa-document-peopleToNotify-' . $i . '-name-last',
+                    $personToNotify->name->last
+                );
+                $this->setData(
+                    'lpa-document-peopleToNotify-' . $i . '-address-address1',
+                    $personToNotify->address->address1
+                );
+                $this->setData(
+                    'lpa-document-peopleToNotify-' . $i . '-address-address2',
+                    $personToNotify->address->address2
+                );
+                $this->setData(
+                    'lpa-document-peopleToNotify-' . $i . '-address-address3',
+                    $personToNotify->address->address3
+                );
+                $this->setData(
+                    'lpa-document-peopleToNotify-' . $i . '-address-postcode',
+                    $personToNotify->address->postcode
+                );
             } else {
-                //  Add a strike through on the appropriate page
+                // Add a strike through on the appropriate page
                 $this->addStrikeThrough('people-to-notify-' . $i, 7);
             }
         }
 
         if (count($peopleToNotify) > self::MAX_PEOPLE_TO_NOTIFY_SECTION_6) {
-            //  TODO - Historic bug - the check box on the H&W PDF is named incorrecly
+            // TODO - Historic bug - the check box on the H&W PDF is named incorrecly
```

Comment:
How old is this?

Reply:
This refers to a minor issue with a checkbox on the HW PDF having the wrong name, which this works around. It would require a change to the PDF to fix, which is not worth the hassle. I expanded the comment to better explain what's going on.